File: Programming-languages-and-laboratory/Python/Exercises/07_compito_java_20160129/Cliente.py
class Cliente:
    def __init__(self, tipo, codice, indirizzo, giorno, mese, anno):
        self._tipo = tipo
        self._codice = codice
        self._indirizzo = indirizzo
        self._giorno = giorno
        self._mese = mese
        self._anno = anno
        self._premi_pagati = []
        self._totale_premi_pagati = 0

    def add_premio(self, p):
        self._premi_pagati.append(p)
        self._totale_premi_pagati += p
        
    def get_premio(self):
        return self._totale_premi_pagati

    def __str__(self):
        return str(self._codice)

File: II - Linguaggi di programmazione e laboratorio/Esame/Java/07_compito_java_2016_01_29/Cliente.java
public class Cliente {
    protected int codice;
    protected String indirizzo;
    protected int giorno, mese, anno;

    public Cliente(int codice, String indirizzo, int giorno, int mese, int anno) {
        this.codice = codice;
        this.indirizzo = indirizzo;
        this.giorno = giorno;
        this.mese = mese;
        this.anno = anno;
    }
}